erpnext/hr/doctype/full_and_final_statement/full_and_final_statement.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Full And Final Statement', {
+	onload: function(frm) {
+		frm.events.set_queries(frm, "payables");
+		frm.events.set_queries(frm, "receivables");
+	},
+
+	set_queries: function(frm, type) {
+		frm.set_query('reference_document_type', type, function () {
+			let modules = ["HR", "Payroll", "Loan Management"];
+			return {
+				filters: {
+					istable: 0,
+					issingle: 0,
+					module: ["In", modules]
+				}
+			};
+		});
+	},
+
+	refresh: function(frm) {
+		if (frm.doc.docstatus == 1 && frm.doc.status == "Unpaid") {
```

Comment:
After Creating Journal Entry The status of Docs will Change From Unpaid to Paid. HR needs to check the Documents, amount and have to mark it **settled** Before Creating the JV, It will Avoid **Wrong Payment** to employees if the HR User/Manager Mark It Stteled Manually. Open For better Workflow Suggestions.